.github/workflows/ci.yaml

```diff
@@ -508,4 +508,38 @@ jobs:
         run: |
           . venv/bin/activate
           pip install -e .
-          pytest -m primer_stdlib -n auto
+          pytest -m primer -n auto -k test_primer_stdlib_no_crash
+
+  pytest-primer-external:
+    name: Run primer tests on external libs Python ${{ matrix.python-version }} (Linux)
+    runs-on: ubuntu-latest
+    needs: prepare-tests-linux
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9, "3.10"]
+    steps:
+      - name: Check out code from GitHub
+        uses: actions/checkout@v2.3.5
+      - name: Set up Python ${{ matrix.python-version }}
+        id: python
+        uses: actions/setup-python@v2.2.2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Restore Python virtual environment
+        id: cache-venv
+        uses: actions/cache@v2.1.6
+        with:
+          path: venv
+          key:
+            ${{ runner.os }}-${{ steps.python.outputs.python-version }}-${{
+            needs.prepare-tests-linux.outputs.python-key }}
+      - name: Fail job if Python cache restore failed
+        if: steps.cache-venv.outputs.cache-hit != 'true'
+        run: |
+          echo "Failed to restore Python venv from cache"
+          exit 1
+      - name: Run pytest
+        run: |
+          . venv/bin/activate
+          pip install -e .
+          pytest -m primer -n auto -k test_primer_external_packages_no_crash
```

Comment:
`pytest` isn't quite ideal as it only displays the test output if it fails. Could be difficult to detect regressions, at lot a new false-positives, that way.

Reply:
Maybe the `--capture=tee-sys` option works?
https://pytest.org/en/6.2.x/capture.html#setting-capturing-methods-or-disabling-capturing

Reply:
We're going to handle new false positives later on, right now we're just checking for fatal messages and crashes.

Reply:
https://github.com/PyCQA/pylint/issues/5364